joomla/joomla-cms
# [4.0] Replace com_content 'Image Float' with 'Image Class"

Pull Request for Issue #17399 .

### Summary of Changes
Replaces the 'Image Float ' field in com_contents with an 'Image Class' field (For reasons why see the original issue.. #17399)


### Testing Instructions
Navigate to the **Images and Links** tab in article edit. Set an image and add a utility class to the **Image Class** field (eg. float-right). Check frontend and ensure class has been applied to the image (eg. image floated to right / inspect element).
